src/test/integration/IntegrationBase.t.sol

```diff
@@ -349,7 +354,11 @@ abstract contract IntegrationBase is IntegrationDeployer {
         uint[] memory balances = new uint[](tokens.length);
 
         for (uint i = 0; i < tokens.length; i++) {
-            balances[i] = tokens[i].balanceOf(address(staker));
+            if (address(tokens[i]) == address(0)) {
+                balances[i] = address(staker).balance;
```

Comment:
does using the zero address for native ETH make sense here / could it be confusing? maybe we want to use the 0xbeaC0eeEeeeeEEeEeEEEEeeEEeEeeeEeeEEBEaC0 we're using elsewhere?
not too sure or strongly opinionated

Reply:
On board with using `0xbeaC0..`

Reply:
Funny, as it turns out I already had a `NATIVE_ETH` constant declared that did this - I just wasn't using it!

Good call, will make this fix :smile:

Reply:
[3eba0cc](https://github.com/Layr-Labs/eigenlayer-contracts/pull/340/commits/3eba0cc4f8f1c459995f6e857cdc7cffc0c36215)